model/riscv_sys_control.sail

```diff
@@ -134,6 +141,12 @@ function is_CSR_defined (csr : csreg, p : Privilege) -> bool =
     /* disabled trigger/debug module */
     0x7a0 => p == Machine,
 
+    /* Smcntrpmf */
+    0x321 => p == Machine, // mcyclecfg
+    0x322 => p == Machine, // minstretcfg
```

Comment:
This block could be placed between `mip` and `pmpcfg0` to keep the list sorted by CSR address.